test/functional/test_framework/test_node.py

```diff
@@ -277,7 +286,7 @@ def wait_for_rpc_connection(self):
                     pass  # Port not yet open?
                 else:
                     raise  # unknown OS error
-            except ValueError as e:  # cookie file not found and no rpcuser or rpcpassword; bitcoind is still starting
+            except ValueError:  # cookie file not found and no rpcuser or rpcpassword; bitcoind is still starting
                 if "No RPC credentials" not in str(e):
```

Comment:
This will throw an error because `e` is not defined. For some reason you seem to have removed it from the `except` clause.